components/centraldashboard/app/api_workgroup.ts

```diff
@@ -181,9 +181,18 @@ export class WorkgroupApi {
         const namespaces = mapWorkgroupBindingToSimpleBinding(
             bindings.body.bindings || []
         );
+        // Ensure we do not return duplicate namespaces.
+        const duplicateNamespaces: string[] = [];
+        const distinctNamespaces = namespaces.reduce((acc, val) => {
```

Comment:
This fixes (I think) https://zbrt.atl.zillow.net/browse/AIP-7008 need to sync with Max on this one as I'd forgotten we'd actually assigned this to him.